assets/chezmoi.io/docs/user-guide/password-managers/1password.md

```diff
@@ -6,22 +6,24 @@ expose data as a template function.
 
 !!! note
 
-    [1Password CLI 2.0](https://developer.1password.com/) has been released.
+    The[1Password CLI 2.0](https://developer.1password.com/) has been released.
```

Comment:
There’s a space missing after The, here.

Reply:
Thanks! Fixed with f86b5b8551c7688a089f1b801dee27ac30685c35.